docs/serving/samples/hello-world/helloworld-kotlin/README.md

```diff
@@ -109,7 +109,7 @@ recreate the source files from this folder.
    ```docker
    # Use the official gradle image to create a build artifact.
    # https://hub.docker.com/_/gradle
-   FROM gradle as builder
+   FROM gradle:4.10 as builder
```

Comment:
Same question about a specific version of gradle here.

Reply:
same here